gtest_parallel_tests.py

```diff
@@ -393,10 +393,10 @@ def _call_find_tests(self, test_data, options=None):
         test_data.keys(), [], options, TestTimesMock(self, test_data))
     return tasks, popen_mock
 
-  def test_tasks_are_sorted(self):
+  def test_tasks_are_sorted_in_descending_order(self):
     tasks, _ = self._call_find_tests(
         self.MULTIPLE_BINARIES_MULTIPLE_TESTS_ONE_FAILURE)
-    self.assertEqual(tasks, sorted(tasks))
+    self.assertEqual(tasks, sorted(tasks, reverse=True))
```

Comment:
Now that I think about it, we should also do something like:
```
self.assertEqual([task.last_execution_time for task in tasks],
                 [None, 4, 4, 3, 2])
```
to make sure tasks are actually sorted as we want them to.

Reply:
Agreed, PTAL.